Misc/NEWS.d/next/Tests/2019-09-25-12-18-31.bpo-38270._x-9uH.rst

```diff
@@ -0,0 +1,3 @@
+test.support now has a helper function to check for availibility of a hash
+digest function. Several tests are refactored to either avoid MD5 or to mark
```

Comment:
You may mention that some tests are updated to use SHA256 rather than MD5.

Reply:
Done